packages/pipeline/src/scripts/pull_competing_dex_trades.ts

```diff
@@ -39,14 +44,29 @@ async function getAndSaveTradesAsync(): Promise<void> {
 }
 
 async function getLastSeenTimestampAsync(tradesRepository: Repository<DexTrade>): Promise<number> {
+    const hoursPerDay = 24;
+    const minutesPerHour = 60;
+    const secondsPerMinute = 60;
+    const millisecondsPerSecond = 1000;
+    const millisecondsPerDay = millisecondsPerSecond * secondsPerMinute * minutesPerHour * hoursPerDay;
+
+    const oldestTimestampWilling = Date.now() - MAX_DAYS * millisecondsPerDay;
+
     if ((await tradesRepository.count()) === 0) {
-        return 0;
+        return oldestTimestampWilling;
```

Comment:
does `oldestTimestampWilling` have a special meaning here? why not `oldestFetchableTimestamp`?

Reply:
Because it's not the oldest one that can be fetched; in theory we can fetch arbitrarily old data.  I said "willing" because it's the oldest timestamp we're willing to go back to, per the MAX_DAYS constant defined above.